draft-linker-adem-wg-adem-core.md

```diff
@@ -349,6 +363,10 @@ The certificate authenticating that website MUST be valid for the OI and all the
 * For each root public key with kid `<KID>` (to be understood as a placeholder): `<KID>.adem-configuration.<OI>`
 Beyond these subdomains, `adem-configuration.<OI>` MUST NOT have further subdomains.
 
+TODO(djackson) - Why have you gone with subdomains over a folder in /.well-known/? 
+               - Oh wait - for the certificate transparency benefit? 
+               - My thinking was that the subdomain would not actually need to correspond to a website, just be included on the certificate. 
```

Comment:
Exactly, for the CT benefit. It's surely debatable if it needs to correspond to a website. I thought, delivering the root keys over these related subdomains yields a cleaner design. At the end of the day, `/.well-known/` is just a suffix, isn't it?